pypeit/wavecalib.py

```diff
@@ -134,12 +130,61 @@ def _build_wv_calib(self, method, skip_QA=False):
 
         # Obtain calibration for all slits
         if method == 'simple':
+            # Should only run this on 1 slit
+            self.arcparam['n_first'] = 2
+            self.arcparam['n_final'] = 3
+            self.arcparam['func'] = 'legendre'
+            self.arcparam['nsig_rej'] = 2.
+            self.arcparam['nsig_rej_final'] = 3.
+            self.arcparam['match_toler'] = 3.
+            self.arcparam['disp_toler'] = 0.1
+            self.arcparam['Nstrong'] = 13
+
+            CuI = waveio.load_line_list('CuI', use_ion=True, NIST=True)
+            ArI = waveio.load_line_list('ArI', use_ion=True, NIST=True)
+            ArII = waveio.load_line_list('ArII', use_ion=True, NIST=True)
+            llist = vstack([CuI, ArI, ArII])
+            self.arcparam['llist'] = llist
+
             self.wv_calib = arc.simple_calib_driver(self.msarc, self.arcparam, self.arccen, ok_mask,
                                                     nfitpix=self.par['nfitpix'],
                                                     IDpixels=self.par['IDpixels'],
                                                     IDwaves=self.par['IDwaves'])
         elif method == 'arclines':
-            self.wv_calib = arc.calib_with_arclines(self.arcparam, self.arccen, ok_mask=ok_mask)
+            #self.wv_calib = arc.calib_with_arclines(self.arcparam, self.arccen, ok_mask=ok_mask)
```

Comment:
I'm not sure what you are doing with the logic here, but these are clearly GMOS specific hacks that should not be checked into the master branch.

Reply:
there are no hacks on the `general` method which what
everything uses.  And given semi-brute is broken these do nothing.